### DISCUSSION TITLE
Felt, leather, cork, tweed ... cover instead of coating

### DISCUSSION THREAD

**Discussion by gerdb** [cardonabits/haxo-hw#40] [Show and tell]
Instead of coating the PCB I used a felt cover. Here are some pictures and the pattern I used for my v1.0.1 PCB. I cut it with a sharp single side razor blade:
![haxophone2](https://github.com/cardonabits/haxo-hw/assets/883695/83ce625a-0d85-4f03-ba34-7f407900c2f3)
![haxophone3](https://github.com/cardonabits/haxo-hw/assets/883695/ce887348-ec62-4123-8878-276c66c09b8a)
![haxophone4](https://github.com/cardonabits/haxo-hw/assets/883695/188b5f4e-ee39-43be-a883-f967c4a9f300)
![haxophone5](https://github.com/cardonabits/haxo-hw/assets/883695/d46a0b83-9111-4814-9f27-e13edc1bae54)
[haxophone_stencil.pdf](https://github.com/cardonabits/haxo-hw/files/12897770/haxophone_stencil.pdf)

**Comment by jcard0na** [comment]
This is absolutely brilliant!  I'm totally upgrading my own haxophone today!  Eco friendly, beautiful and pleasant to the touch.
Does felt stay in place, or you had to glue it?  The PCB is sensitive to [leakage paths](https://github.com/cardonabits/haxo-hw/blob/main/docs/assembly.md#general-advice), so we would have make sure the felt/wool does not absorb moisture.

Thanks for sharing this!!

**Comment by gerdb** [comment]
I only had to glue the 2 parts on the side and the one over the RPI. There are also CNC/laser services that cut felt or cork. I've visited such a company that cuts cork in Porto some years ago. :smiley:
In any case I would suggest to add 3 additional pull-up resistors (1k) to COL0,1 and 2 to increase the robustness.

**Reply by jcard0na** [reply]
I was on the fence about external pull-ups, but you've helped me decide.  I've just committed this change
![image](https://github.com/cardonabits/haxo-hw/assets/676181/fab22502-80c4-437c-8a17-dca868696503)
28a57dc0c2ac25278607cefc5118acb0723f45b3

I'm going with 10K resistors instead of the 1K you recommend.  I believe this is a good compromise that protects us against ~250 μA of leakage current, while keeping current consumption reasonably low.  I'm also using large footprints (0805) to make it easy for people to replace and/or remove.

Thanks!
